docker/indexer/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM quay.io/apibara/sink-mongo:0.7.0 as indexer
+
+FROM debian:bookworm as apibara-build
+RUN apt-get update && apt-get install -y patchelf && rm -rf /var/lib/apt/lists/*
```

Comment:
Here:
- Maybe we can use the `--no-install-recommends` tag in order to avoid installation of recommended packages that are useless for us no?
- Maybe we can add here an `apt-get clean` instruction to clean stuffs and lighten the final image?

Reply:
Let us keep as is :)